matching-notification-rest/src/main/java/org/phenotips/matchingnotification/rest/internal/DefaultMatchingNotificationResource.java

```diff
@@ -343,6 +343,9 @@ private Response getMatchesResponse(@Nullable final double score, @Nullable fina
             matchesJson = buildMatchesJSONArray(matches);
             matchesJson.put(REQ_NO, reqNo);
         }
+        matchesJson.put("minScore", score);
+        matchesJson.put("minPhenotipicScore", phenScore);
+        matchesJson.put("minGenotipicScore", genScore);
```

Comment:
Please add `fromDate` and `toDate` parameters to the response if their values are meaningful.

Reply:
Now that I see the returned JSON I would like to suggest some structure improvements:

```
{
  reqNo : <number>,
  parameters : {
    minScore: ...,
    minPhenScore : ...,
    minGenScore : ...,
    fromDate : ...,
    toDate: ...
  },
  dateGenerated: 'yyyy-mm-ddTHH:MM:ssZ',
  results : [...],
  returnedCount : ...
}
```

Basically the changes are:
* Group all parameters provided by the user under `parameters`
* include a top-level field that specifies when the data was generated